api/src/main/java/org/openmrs/module/fhir2/api/FhirMedicationRequestService.java

```diff
@@ -13,8 +13,8 @@
 
 import org.hl7.fhir.r4.model.MedicationRequest;
 
-public interface FhirMedicationRequestService {
-	
-	MedicationRequest getMedicationRequestByUuid(@NotNull String uuid);
+public interface FhirMedicationRequestService extends FhirService<MedicationRequest>{
+	@Override
```

Comment:
Can you add a blank new line before this?

Reply:
fixed it to